Are you taking term requests? I have several related to radiometry.

watt meter^-2 micrometer^-1. 
unit of spectral emittance, spectral irradiance 
existing scaled: W-PER-M2-NanoM, W-PER-M2-M, MilliW-PER-M2-NanoM (all currently a measurement unit of unknown)

watt meter^-2 micrometer^-1 steradian^-1
microwatt centimeter^-2 micrometer^-1 steradian^-1 
unit of spectral radiance 
existing scaled: W-PER-M2-NanoM-SR, MilliW-PER-M2-NanoM-SR (both currently a measurement unit of unknown)

Joule meter^-2 second^.5 kelvin -1
unit of thermal inertia

Reference
https://nsgreg.nga.mil/doc/view?i=5613 Vol 1-App X BANDSB Spectral Data Quantities and Units

I'm not a sophisticated user of QUDT. I'm using QUDT units in the hope that a group of ontologies, of which I am writing one, all use the same unit ontology. When I use units, I use the QUDT IRI and extract just what I need (which is mostly label and description - not yet sure about quantity kinds). I don't do conversions. Perhaps the group will need that at some point, but not at the moment. So I'm not sure how much I can help with testing. I also work in OWL.

I looked at the submission guide. FYI the link in "All possible properties that could be included can be found in the QUDT Schema file [/schema/SCHEMA_QUDT-v2.1.ttl](https://github.com/qudt/qudt-public-repo/schema/SCHEMA_QUDT-v2.1.ttl) in this repository." is broken.

The naming stuff seems clear enough. I had surmised enough of the rules applicable to my units by looking at other units.

On [Introducing Dimension Vectors for SI Units](https://github.com/qudt/qudt-public-repo/wiki/User-Guide-for-QUDT#3-introducing-dimension-vectors-for-si-units) hasDimensionVector is a property of quantity kind in one example, and a property of unit in another. I'm guessing it should only be on unit? It doesn't seem to be a requirement that all units of a quantity kind have to have the same dimension vector. It isn't the case for my case of thermal inertia.

I cloned the repo and ran ```mvn clean install``` as instructed in the contributor installation guide, but got an error, included below.

I had a look at the PR. Looks like good stuff. The 'isScalingOf' property would be useful. It wasn't clear to me what factorUnit is doing. Including an definition in the PR would be helpful.

I'm attaching what I've got so far. It's not a small amount of work for a contributor who has a unit or two and is not familiar with QUDT in detail. Doing a fork, finding the right files, etc is even more work. I know it's work for somebody, but it would be a lot friendlier to have a term request form. Hopefully I got the details right, but perhaps you can check.

[qudt-submission.txt](https://github.com/user-attachments/files/19045344/qudt-submission.txt)

Its text because ttl isn't an allowable suffix for attachments.

```
[ERROR] Step 'RdfFormatter' found problem in 'src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl':
Error formatting file /Users/alanr/repos/qudt-public-repo/src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl
java.lang.RuntimeException: Error formatting file /Users/alanr/repos/qudt-public-repo/src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl
    at com.diffplug.spotless.rdf.RdfFormatterFunc.apply (RdfFormatterFunc.java:79)
    at com.diffplug.spotless.FormatterStepEqualityOnStateSerialization.format (FormatterStepEqualityOnStateSerialization.java:48)
    at com.diffplug.spotless.Formatter.compute (Formatter.java:244)
    at com.diffplug.spotless.PaddedCell.calculateDirtyState (PaddedCell.java:203)
    at com.diffplug.spotless.PaddedCell.calculateDirtyState (PaddedCell.java:190)
    at com.diffplug.spotless.maven.SpotlessCheckMojo.process (SpotlessCheckMojo.java:81)
    at com.diffplug.spotless.maven.AbstractSpotlessMojo.execute (AbstractSpotlessMojo.java:256)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:972)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
Caused by: java.lang.IllegalStateException: Formatted RDF is not isomorphic with original, which means that formatting changed the data.
This could be a bug in the formatting system leading to data corruption and should be reported. 
If you are not scared to lose data, you can disable this check by setting the config option 'verify' to 'false'

Diff:
< 40,179 triples> 40,178 triples
    at com.diffplug.spotless.rdf.RdfFormatterFunc.veryfyResult (RdfFormatterFunc.java:121)
    at com.diffplug.spotless.rdf.RdfFormatterFunc.formatTurtle (RdfFormatterFunc.java:102)
    at com.diffplug.spotless.rdf.RdfFormatterFunc.apply (RdfFormatterFunc.java:64)
    at com.diffplug.spotless.FormatterStepEqualityOnStateSerialization.format (FormatterStepEqualityOnStateSerialization.java:48)
    at com.diffplug.spotless.Formatter.compute (Formatter.java:244)
    at com.diffplug.spotless.PaddedCell.calculateDirtyState (PaddedCell.java:203)
    at com.diffplug.spotless.PaddedCell.calculateDirtyState (PaddedCell.java:190)
    at com.diffplug.spotless.maven.SpotlessCheckMojo.process (SpotlessCheckMojo.java:81)
    at com.diffplug.spotless.maven.AbstractSpotlessMojo.execute (AbstractSpotlessMojo.java:256)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:972)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:07 min
[INFO] Finished at: 2025-03-02T16:17:05-05:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.diffplug.spotless:spotless-maven-plugin:2.44.0.BETA3:check (check-source-format) on project qudt-public-repo: Unable to check file /Users/alanr/repos/qudt-public-repo/src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl: Error formatting file /Users/alanr/repos/qudt-public-repo/src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl: Formatted RDF is not isomorphic with original, which means that formatting changed the data.
[ERROR] This could be a bug in the formatting system leading to data corruption and should be reported. 
[ERROR] If you are not scared to lose data, you can disable this check by setting the config option 'verify' to 'false'
[ERROR] 
[ERROR] Diff:
[ERROR] < 40,179 triples> 40,178 triples
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

You are right, a term request form is a good idea. Building the form,
deploying it, gathering the results, writing them into an RDF graph, and
submitting that graph as a PR would be time consuming on a level that
hasn't found the kind of manpower we currently have. Our priority has been
to stabilize and make consistent the vocabulary we have, and you have
pointed out some issues that we can/will address.

Adding units to the vocabulary is not something I would recommend for non
ontologists but it kind of begs the question of what use someone has for an
ontology / vocabulary if they aren't familiar with same. I do not want to
make assumptions about who uses QUDT or what their background is, but to
date we haven't obfuscated the ontology to a point where someone that wants
a unit dictionary but not the semantic model would find it useful. Unit
conversion and dimensional analysis apps have been prototyped for QUDT but
other uses (e.g., engineering) have required deep knowledge of OWL and
SPARQL.

Often we can add units into the existing repository if the person of
interest provides the required content from the unit submission guidelines,
but you can just as easily get the units graph, copy a unit and edit it and
then submit the new graph. It is mostly an issue of manpower and that is
why we rely on the people that want to see their units in the vocabulary
going through the effort of submitting the PR. It isn't to be mean by any
means.

On Sun, Mar 2, 2025 at 4:00 PM Alan Ruttenberg ***@***.***>
wrote:

> I'm not a sophisticated user of QUDT. I'm using QUDT units in the hope
> that a group of ontologies, of which I am writing one, all use the same
> unit ontology. When I use units, I use the QUDT IRI and extract just what I
> need (which is mostly label and description - not yet sure about quantity
> kinds). I don't do conversions. Perhaps the group will need that at some
> point, but not at the moment. So I'm not sure how much I can help with
> testing. I also work in OWL.
>
> I looked at the submission guide. FYI the link in "All possible properties
> that could be included can be found in the QUDT Schema file
> /schema/SCHEMA_QUDT-v2.1.ttl
> <https://github.com/qudt/qudt-public-repo/schema/SCHEMA_QUDT-v2.1.ttl> in
> this repository." is broken.
>
> The naming stuff seems clear enough. I had surmised enough of the rules
> applicable to my units by looking at other units.
>
> On Introducing Dimension Vectors for SI Units
> <https://github.com/qudt/qudt-public-repo/wiki/User-Guide-for-QUDT#3-introducing-dimension-vectors-for-si-units>
> hasDimensionVector is a property of quantity kind in one example, and a
> property of unit in another. I'm guessing it should only be on unit? It
> doesn't seem to be a requirement that all units of a quantity kind have to
> have the same dimension vector. It isn't the case for my case of thermal
> inertia.
>
> I cloned the repo and ran mvn clean install as instructed in the
> contributor installation guide, but got an error, included below.
>
> I had a look at the PR. Looks like good stuff. The 'isScalingOf' property
> would be useful. It wasn't clear to me what factorUnit is doing. Including
> an definition in the PR would be helpful.
>
> I'm attaching what I've got so far. It's not a small amount of work for a
> contributor who has a unit or two and is not familiar with QUDT in detail.
> Doing a fork, finding the right files, etc is even more work. I know it's
> work for somebody, but it would be a lot friendlier to have a term request
> form. Hopefully I got the details right, but perhaps you can check.
>
> qudt-submission.txt
> <https://github.com/user-attachments/files/19045344/qudt-submission.txt>
>
> Its text because ttl isn't an allowable suffix for attachments.
>
> [ERROR] Step 'RdfFormatter' found problem in 'src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl':
> Error formatting file /Users/alanr/repos/qudt-public-repo/src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl
> java.lang.RuntimeException: Error formatting file /Users/alanr/repos/qudt-public-repo/src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl
>     at com.diffplug.spotless.rdf.RdfFormatterFunc.apply (RdfFormatterFunc.java:79)
>     at com.diffplug.spotless.FormatterStepEqualityOnStateSerialization.format (FormatterStepEqualityOnStateSerialization.java:48)
>     at com.diffplug.spotless.Formatter.compute (Formatter.java:244)
>     at com.diffplug.spotless.PaddedCell.calculateDirtyState (PaddedCell.java:203)
>     at com.diffplug.spotless.PaddedCell.calculateDirtyState (PaddedCell.java:190)
>     at com.diffplug.spotless.maven.SpotlessCheckMojo.process (SpotlessCheckMojo.java:81)
>     at com.diffplug.spotless.maven.AbstractSpotlessMojo.execute (AbstractSpotlessMojo.java:256)
>     at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
>     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
>     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
>     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
>     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
>     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
>     at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
>     at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
>     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
>     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
>     at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
>     at org.apache.maven.cli.MavenCli.execute (MavenCli.java:972)
>     at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
>     at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
>     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
>     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
>     at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
>     at java.lang.reflect.Method.invoke (Method.java:568)
>     at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
>     at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
>     at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
>     at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
> Caused by: java.lang.IllegalStateException: Formatted RDF is not isomorphic with original, which means that formatting changed the data.
> This could be a bug in the formatting system leading to data corruption and should be reported.
> If you are not scared to lose data, you can disable this check by setting the config option 'verify' to 'false'
>
> Diff:
> < 40,179 triples> 40,178 triples
>     at com.diffplug.spotless.rdf.RdfFormatterFunc.veryfyResult (RdfFormatterFunc.java:121)
>     at com.diffplug.spotless.rdf.RdfFormatterFunc.formatTurtle (RdfFormatterFunc.java:102)
>     at com.diffplug.spotless.rdf.RdfFormatterFunc.apply (RdfFormatterFunc.java:64)
>     at com.diffplug.spotless.FormatterStepEqualityOnStateSerialization.format (FormatterStepEqualityOnStateSerialization.java:48)
>     at com.diffplug.spotless.Formatter.compute (Formatter.java:244)
>     at com.diffplug.spotless.PaddedCell.calculateDirtyState (PaddedCell.java:203)
>     at com.diffplug.spotless.PaddedCell.calculateDirtyState (PaddedCell.java:190)
>     at com.diffplug.spotless.maven.SpotlessCheckMojo.process (SpotlessCheckMojo.java:81)
>     at com.diffplug.spotless.maven.AbstractSpotlessMojo.execute (AbstractSpotlessMojo.java:256)
>     at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
>     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
>     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
>     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
>     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
>     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
>     at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
>     at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
>     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
>     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
>     at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
>     at org.apache.maven.cli.MavenCli.execute (MavenCli.java:972)
>     at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
>     at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
>     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
>     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
>     at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
>     at java.lang.reflect.Method.invoke (Method.java:568)
>     at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
>     at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
>     at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
>     at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
> [INFO] ------------------------------------------------------------------------
> [INFO] BUILD FAILURE
> [INFO] ------------------------------------------------------------------------
> [INFO] Total time:  01:07 min
> [INFO] Finished at: 2025-03-02T16:17:05-05:00
> [INFO] ------------------------------------------------------------------------
> [ERROR] Failed to execute goal com.diffplug.spotless:spotless-maven-plugin:2.44.0.BETA3:check (check-source-format) on project qudt-public-repo: Unable to check file /Users/alanr/repos/qudt-public-repo/src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl: Error formatting file /Users/alanr/repos/qudt-public-repo/src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl: Formatted RDF is not isomorphic with original, which means that formatting changed the data.
> [ERROR] This could be a bug in the formatting system leading to data corruption and should be reported.
> [ERROR] If you are not scared to lose data, you can disable this check by setting the config option 'verify' to 'false'
> [ERROR]
> [ERROR] Diff:
> [ERROR] < 40,179 triples> 40,178 triples
> [ERROR] -> [Help 1]
> [ERROR]
> [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
> [ERROR] Re-run Maven using the -X switch to enable full debug logging.
> [ERROR]
> [ERROR] For more information about the errors and possible solutions, please read the following articles:
> [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/qudt/qudt-public-repo/issues/1139#issuecomment-2692978732>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AATQRWNRRZ6WXGIL63QWKZT2SOLTFAVCNFSM6AAAAABYBMOKMKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMOJSHE3TQNZTGI>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
> [image: alanruttenberg]*alanruttenberg* left a comment
> (qudt/qudt-public-repo#1139)
> <https://github.com/qudt/qudt-public-repo/issues/1139#issuecomment-2692978732>
>
> I'm not a sophisticated user of QUDT. I'm using QUDT units in the hope
> that a group of ontologies, of which I am writing one, all use the same
> unit ontology. When I use units, I use the QUDT IRI and extract just what I
> need (which is mostly label and description - not yet sure about quantity
> kinds). I don't do conversions. Perhaps the group will need that at some
> point, but not at the moment. So I'm not sure how much I can help with
> testing. I also work in OWL.
>
> I looked at the submission guide. FYI the link in "All possible properties
> that could be included can be found in the QUDT Schema file
> /schema/SCHEMA_QUDT-v2.1.ttl
> <https://github.com/qudt/qudt-public-repo/schema/SCHEMA_QUDT-v2.1.ttl> in
> this repository." is broken.
>
> The naming stuff seems clear enough. I had surmised enough of the rules
> applicable to my units by looking at other units.
>
> On Introducing Dimension Vectors for SI Units
> <https://github.com/qudt/qudt-public-repo/wiki/User-Guide-for-QUDT#3-introducing-dimension-vectors-for-si-units>
> hasDimensionVector is a property of quantity kind in one example, and a
> property of unit in another. I'm guessing it should only be on unit? It
> doesn't seem to be a requirement that all units of a quantity kind have to
> have the same dimension vector. It isn't the case for my case of thermal
> inertia.
>
> I cloned the repo and ran mvn clean install as instructed in the
> contributor installation guide, but got an error, included below.
>
> I had a look at the PR. Looks like good stuff. The 'isScalingOf' property
> would be useful. It wasn't clear to me what factorUnit is doing. Including
> an definition in the PR would be helpful.
>
> I'm attaching what I've got so far. It's not a small amount of work for a
> contributor who has a unit or two and is not familiar with QUDT in detail.
> Doing a fork, finding the right files, etc is even more work. I know it's
> work for somebody, but it would be a lot friendlier to have a term request
> form. Hopefully I got the details right, but perhaps you can check.
>
> qudt-submission.txt
> <https://github.com/user-attachments/files/19045344/qudt-submission.txt>
>
> Its text because ttl isn't an allowable suffix for attachments.
>
> [ERROR] Step 'RdfFormatter' found problem in 'src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl':
> Error formatting file /Users/alanr/repos/qudt-public-repo/src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl
> java.lang.RuntimeException: Error formatting file /Users/alanr/repos/qudt-public-repo/src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl
>     at com.diffplug.spotless.rdf.RdfFormatterFunc.apply (RdfFormatterFunc.java:79)
>     at com.diffplug.spotless.FormatterStepEqualityOnStateSerialization.format (FormatterStepEqualityOnStateSerialization.java:48)
>     at com.diffplug.spotless.Formatter.compute (Formatter.java:244)
>     at com.diffplug.spotless.PaddedCell.calculateDirtyState (PaddedCell.java:203)
>     at com.diffplug.spotless.PaddedCell.calculateDirtyState (PaddedCell.java:190)
>     at com.diffplug.spotless.maven.SpotlessCheckMojo.process (SpotlessCheckMojo.java:81)
>     at com.diffplug.spotless.maven.AbstractSpotlessMojo.execute (AbstractSpotlessMojo.java:256)
>     at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
>     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
>     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
>     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
>     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
>     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
>     at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
>     at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
>     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
>     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
>     at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
>     at org.apache.maven.cli.MavenCli.execute (MavenCli.java:972)
>     at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
>     at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
>     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
>     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
>     at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
>     at java.lang.reflect.Method.invoke (Method.java:568)
>     at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
>     at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
>     at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
>     at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
> Caused by: java.lang.IllegalStateException: Formatted RDF is not isomorphic with original, which means that formatting changed the data.
> This could be a bug in the formatting system leading to data corruption and should be reported.
> If you are not scared to lose data, you can disable this check by setting the config option 'verify' to 'false'
>
> Diff:
> < 40,179 triples> 40,178 triples
>     at com.diffplug.spotless.rdf.RdfFormatterFunc.veryfyResult (RdfFormatterFunc.java:121)
>     at com.diffplug.spotless.rdf.RdfFormatterFunc.formatTurtle (RdfFormatterFunc.java:102)
>     at com.diffplug.spotless.rdf.RdfFormatterFunc.apply (RdfFormatterFunc.java:64)
>     at com.diffplug.spotless.FormatterStepEqualityOnStateSerialization.format (FormatterStepEqualityOnStateSerialization.java:48)
>     at com.diffplug.spotless.Formatter.compute (Formatter.java:244)
>     at com.diffplug.spotless.PaddedCell.calculateDirtyState (PaddedCell.java:203)
>     at com.diffplug.spotless.PaddedCell.calculateDirtyState (PaddedCell.java:190)
>     at com.diffplug.spotless.maven.SpotlessCheckMojo.process (SpotlessCheckMojo.java:81)
>     at com.diffplug.spotless.maven.AbstractSpotlessMojo.execute (AbstractSpotlessMojo.java:256)
>     at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
>     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
>     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
>     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
>     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
>     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
>     at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
>     at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
>     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
>     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
>     at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
>     at org.apache.maven.cli.MavenCli.execute (MavenCli.java:972)
>     at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
>     at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
>     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
>     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
>     at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
>     at java.lang.reflect.Method.invoke (Method.java:568)
>     at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
>     at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
>     at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
>     at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
> [INFO] ------------------------------------------------------------------------
> [INFO] BUILD FAILURE
> [INFO] ------------------------------------------------------------------------
> [INFO] Total time:  01:07 min
> [INFO] Finished at: 2025-03-02T16:17:05-05:00
> [INFO] ------------------------------------------------------------------------
> [ERROR] Failed to execute goal com.diffplug.spotless:spotless-maven-plugin:2.44.0.BETA3:check (check-source-format) on project qudt-public-repo: Unable to check file /Users/alanr/repos/qudt-public-repo/src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl: Error formatting file /Users/alanr/repos/qudt-public-repo/src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl: Formatted RDF is not isomorphic with original, which means that formatting changed the data.
> [ERROR] This could be a bug in the formatting system leading to data corruption and should be reported.
> [ERROR] If you are not scared to lose data, you can disable this check by setting the config option 'verify' to 'false'
> [ERROR]
> [ERROR] Diff:
> [ERROR] < 40,179 triples> 40,178 triples
> [ERROR] -> [Help 1]
> [ERROR]
> [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
> [ERROR] Re-run Maven using the -X switch to enable full debug logging.
> [ERROR]
> [ERROR] For more information about the errors and possible solutions, please read the following articles:
> [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/qudt/qudt-public-repo/issues/1139#issuecomment-2692978732>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AATQRWNRRZ6WXGIL63QWKZT2SOLTFAVCNFSM6AAAAABYBMOKMKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMOJSHE3TQNZTGI>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Jack

--- a/src/main/rdf/vocab/dimensionvectors/VOCAB_QUDT-DIMENSION-VECTORS.ttl
+++ b/src/main/rdf/vocab/dimensionvectors/VOCAB_QUDT-DIMENSION-VECTORS.ttl
@@ -1524,6 +1524,7 @@ qkdv:A0E0L0I0M0H-1T0D0
   qudt:dimensionExponentForThermodynamicTemperature -1 ;
   qudt:dimensionExponentForTime 0 ;
   qudt:dimensionlessExponent 0 ;
+  qudt:hasReferenceQuantityKind: quantitykind:ApparentThermalInertia;
   rdfs:isDefinedBy "<http://qudt.org/$$QUDT_VERSION$$/vocab/dimensionvector>" ;
   rdfs:label "A0E0L0I0M0H-1T0D0" .
 

--- a/src/main/rdf/vocab/dimensionvectors/VOCAB_QUDT-DIMENSION-VECTORS.ttl
+++ b/src/main/rdf/vocab/dimensionvectors/VOCAB_QUDT-DIMENSION-VECTORS.ttl
@@ -4007,7 +4007,7 @@ qkdv:A0E-2L2I0M1H-1T-2.5D0
   qudt:hasReferenceQuantityKind quantitykind:ThermalInertia ;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/dimension> .
 
-qkdv:A0E0L-6I0M1H0T-3D-1
+qkdv:A0E0L-6I0M1H0T-3D1
   a qudt:QuantityKindDimensionVector_SI;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/dimensionvector>;
   qudt:dimensionExponentForAmountOfSubstance "0"^^xsd:float ;
@@ -4017,7 +4017,7 @@ qkdv:A0E0L-6I0M1H0T-3D-1
   qudt:dimensionExponentForMass "1"^^xsd:float ;
   qudt:dimensionExponentForThermodynamicTemperature "0"^^xsd:float ;
   qudt:dimensionExponentForTime "-3"^^xsd:float ;
-  qudt:dimensionlessExponent "-1"^^xsd:float ;
+  qudt:dimensionlessExponent "1"^^xsd:float ;
   qudt:hasReferenceQuantityKind quantitykind:SpectralRadiance ;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/dimension> .
 

--- a/src/main/rdf/vocab/dimensionvectors/VOCAB_QUDT-DIMENSION-VECTORS.ttl
+++ b/src/main/rdf/vocab/dimensionvectors/VOCAB_QUDT-DIMENSION-VECTORS.ttl
@@ -4009,23 +4009,8 @@ qkdv:A0E-2L2I0M1H-1T-2.5D0
   qudt:hasReferenceQuantityKind quantitykind:ThermalInertia ;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/dimension> .
 
-qkdv:A0E0L-6I0M1H0T-3D0
-  a qudt:QuantityKindDimensionVector_SI;
-  rdfs:label "A0E0L-6I0M1H0T-3D0";
-  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/dimensionvector>;
-  qudt:dimensionExponentForAmountOfSubstance 0 ;
-  qudt:dimensionExponentForElectricCurrent 0 ;
-  qudt:dimensionExponentForLength -6 ;
-  qudt:dimensionExponentForLuminousIntensity 0 ;
-  qudt:dimensionExponentForMass 1 ;
-  qudt:dimensionExponentForThermodynamicTemperature 0 ;
-  qudt:dimensionExponentForTime -3 ;
-  qudt:dimensionlessExponent 0 ;
-  qudt:hasReferenceQuantityKind quantitykind:SpectralRadiance ;
-  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/dimension> .
-
 qkdv:A0E0L-4I0M1H0T-3D0
-a qudt:QuantityKindDimensionVector_SI;
+  a qudt:QuantityKindDimensionVector_SI;
   rdfs:label "A0E0L-4I0M1H0T-3D0";
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/dimensionvector>;
   qudt:dimensionExponentForAmountOfSubstance 0 ;

--- a/src/main/rdf/vocab/dimensionvectors/VOCAB_QUDT-DIMENSION-VECTORS.ttl
+++ b/src/main/rdf/vocab/dimensionvectors/VOCAB_QUDT-DIMENSION-VECTORS.ttl
@@ -1849,7 +1849,7 @@ qkdv:A0E0L0I0M1H0T-2D0
   rdfs:isDefinedBy <http://qudt.org/$$QUDT_VERSION$$/vocab/dimensionvector> ;
   rdfs:label "A0E0L0I0M1H0T-2D0" .
 
-qkdv:A0E0L0I0M1H0T-3D-1
+qkdv:A0E0L0I0M1H0T-3D1
   a qudt:QuantityKindDimensionVector_CGS, qudt:QuantityKindDimensionVector_ISO, qudt:QuantityKindDimensionVector_Imperial, qudt:QuantityKindDimensionVector_SI ;
   qudt:dimensionExponentForAmountOfSubstance 0 ;
   qudt:dimensionExponentForElectricCurrent 0 ;
@@ -1861,7 +1861,7 @@ qkdv:A0E0L0I0M1H0T-3D-1
   qudt:dimensionlessExponent -1 ;
   qudt:latexDefinition "$U^-1 M T^-3$"^^qudt:LatexString ;
   rdfs:isDefinedBy <http://qudt.org/$$QUDT_VERSION$$/vocab/dimensionvector> ;
-  rdfs:label "A0E0L0I0M1H0T-3D-1" .
+  rdfs:label "A0E0L0I0M1H0T-3D1" .
 
 qkdv:A0E0L0I0M1H0T-3D0
   a qudt:QuantityKindDimensionVector_CGS, qudt:QuantityKindDimensionVector_ISO, qudt:QuantityKindDimensionVector_Imperial, qudt:QuantityKindDimensionVector_SI ;
@@ -4005,7 +4005,6 @@ qkdv:A0E-2L2I0M1H-1T-2.5D0
   qudt:dimensionExponentForTime "-2.5"^^xsd:float ;
   qudt:dimensionlessExponent "0"^^xsd:float ;
   qudt:hasReferenceQuantityKind quantitykind:ThermalInertia ;
-  qudt:isDimensionInSystem qudt:SOQ_ISQ ;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/dimension> .
 
 qkdv:A0E0L-6I0M1H0T-3D-1
@@ -4020,7 +4019,6 @@ qkdv:A0E0L-6I0M1H0T-3D-1
   qudt:dimensionExponentForTime "-3"^^xsd:float ;
   qudt:dimensionlessExponent "-1"^^xsd:float ;
   qudt:hasReferenceQuantityKind quantitykind:SpectralRadiance ;
-  qudt:isDimensionInSystem qudt:SOQ_ISQ ;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/dimension> .
 
 qkdv:A0E0L-4I0M1H0T-3D0
@@ -4034,8 +4032,7 @@ a qudt:QuantityKindDimensionVector_SI;
   qudt:dimensionExponentForThermodynamicTemperature "0"^^xsd:float ;
   qudt:dimensionExponentForTime "-3"^^xsd:float ;
   qudt:dimensionlessExponent "0"^^xsd:float ;
-  qudt:hasReferenceQuantityKind quantitykind:spectralIrradiance , quantitykind:spectralEmittance;
-  qudt:isDimensionInSystem qudt:SOQ_ISQ ;
+  qudt:hasReferenceQuantityKind quantitykind:SpectralIrradiance , quantitykind:SpectralEmittance;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/dimension> .
 
 vaem:GMD_QUDT-DIMENSION-VECTORS

--- a/src/main/rdf/vocab/dimensionvectors/VOCAB_QUDT-DIMENSION-VECTORS.ttl
+++ b/src/main/rdf/vocab/dimensionvectors/VOCAB_QUDT-DIMENSION-VECTORS.ttl
@@ -1524,7 +1524,7 @@ qkdv:A0E0L0I0M0H-1T0D0
   qudt:dimensionExponentForThermodynamicTemperature -1 ;
   qudt:dimensionExponentForTime 0 ;
   qudt:dimensionlessExponent 0 ;
-  qudt:hasReferenceQuantityKind: quantitykind:ApparentThermalInertia;
+  qudt:hasReferenceQuantityKind quantitykind:ApparentThermalInertia;
   rdfs:isDefinedBy "<http://qudt.org/$$QUDT_VERSION$$/vocab/dimensionvector>" ;
   rdfs:label "A0E0L0I0M0H-1T0D0" .
 

--- a/src/main/rdf/vocab/dimensionvectors/VOCAB_QUDT-DIMENSION-VECTORS.ttl
+++ b/src/main/rdf/vocab/dimensionvectors/VOCAB_QUDT-DIMENSION-VECTORS.ttl
@@ -3993,6 +3993,51 @@ qkdv:NotApplicable
   rdfs:isDefinedBy <http://qudt.org/$$QUDT_VERSION$$/vocab/dimensionvector> ;
   rdfs:label "Not Applicable" .
 
+qkdv:A0E-2L2I0M1H-1T-2.5D0
+  a qudt:QuantityKindDimensionVector_SI;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/dimensionvector>;
+  qudt:dimensionExponentForAmountOfSubstance "0"^^xsd:float ;
+  qudt:dimensionExponentForElectricCurrent "-2"^^xsd:float ;
+  qudt:dimensionExponentForLength "2"^^xsd:float ;
+  qudt:dimensionExponentForLuminousIntensity "0"^^xsd:float ;
+  qudt:dimensionExponentForMass "1"^^xsd:float ;
+  qudt:dimensionExponentForThermodynamicTemperature "-1"^^xsd:float ;
+  qudt:dimensionExponentForTime "-2.5"^^xsd:float ;
+  qudt:dimensionlessExponent "0"^^xsd:float ;
+  qudt:hasReferenceQuantityKind quantitykind:ThermalInertia ;
+  qudt:isDimensionInSystem qudt:SOQ_ISQ ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/dimension> .
+
+qkdv:A0E0L-6I0M1H0T-3D-1
+  a qudt:QuantityKindDimensionVector_SI;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/dimensionvector>;
+  qudt:dimensionExponentForAmountOfSubstance "0"^^xsd:float ;
+  qudt:dimensionExponentForElectricCurrent "0"^^xsd:float ;
+  qudt:dimensionExponentForLength "-6"^^xsd:float ;
+  qudt:dimensionExponentForLuminousIntensity "0"^^xsd:float ;
+  qudt:dimensionExponentForMass "1"^^xsd:float ;
+  qudt:dimensionExponentForThermodynamicTemperature "0"^^xsd:float ;
+  qudt:dimensionExponentForTime "-3"^^xsd:float ;
+  qudt:dimensionlessExponent "-1"^^xsd:float ;
+  qudt:hasReferenceQuantityKind quantitykind:SpectralRadiance ;
+  qudt:isDimensionInSystem qudt:SOQ_ISQ ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/dimension> .
+
+qkdv:A0E0L-4I0M1H0T-3D0
+a qudt:QuantityKindDimensionVector_SI;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/dimensionvector>;
+  qudt:dimensionExponentForAmountOfSubstance "0"^^xsd:float ;
+  qudt:dimensionExponentForElectricCurrent "0"^^xsd:float ;
+  qudt:dimensionExponentForLength "-4"^^xsd:float ;
+  qudt:dimensionExponentForLuminousIntensity "0"^^xsd:float ;
+  qudt:dimensionExponentForMass "1"^^xsd:float ;
+  qudt:dimensionExponentForThermodynamicTemperature "0"^^xsd:float ;
+  qudt:dimensionExponentForTime "-3"^^xsd:float ;
+  qudt:dimensionlessExponent "0"^^xsd:float ;
+  qudt:hasReferenceQuantityKind quantitykind:spectralIrradiance , quantitykind:spectralEmittance;
+  qudt:isDimensionInSystem qudt:SOQ_ISQ ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/dimension> .
+
 vaem:GMD_QUDT-DIMENSION-VECTORS
   a vaem:GraphMetaData ;
   dcterms:contributor "Jack Hodges" ;

--- a/src/main/rdf/vocab/dimensionvectors/VOCAB_QUDT-DIMENSION-VECTORS.ttl
+++ b/src/main/rdf/vocab/dimensionvectors/VOCAB_QUDT-DIMENSION-VECTORS.ttl
@@ -3996,6 +3996,7 @@ qkdv:NotApplicable
 qkdv:A0E-2L2I0M1H-1T-2.5D0
   a qudt:QuantityKindDimensionVector_SI;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/dimensionvector>;
+  rdfs:label "A0E-2L2I0M1H-1T-2.5D0" ;
   qudt:dimensionExponentForAmountOfSubstance "0"^^xsd:float ;
   qudt:dimensionExponentForElectricCurrent "-2"^^xsd:float ;
   qudt:dimensionExponentForLength "2"^^xsd:float ;
@@ -4009,6 +4010,7 @@ qkdv:A0E-2L2I0M1H-1T-2.5D0
 
 qkdv:A0E0L-6I0M1H0T-3D0
   a qudt:QuantityKindDimensionVector_SI;
+  rdfs:label "A0E0L-6I0M1H0T-3D0";
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/dimensionvector>;
   qudt:dimensionExponentForAmountOfSubstance "0"^^xsd:float ;
   qudt:dimensionExponentForElectricCurrent "0"^^xsd:float ;
@@ -4023,6 +4025,7 @@ qkdv:A0E0L-6I0M1H0T-3D0
 
 qkdv:A0E0L-4I0M1H0T-3D0
 a qudt:QuantityKindDimensionVector_SI;
+  rdfs:label "A0E0L-4I0M1H0T-3D0";
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/dimensionvector>;
   qudt:dimensionExponentForAmountOfSubstance "0"^^xsd:float ;
   qudt:dimensionExponentForElectricCurrent "0"^^xsd:float ;

--- a/src/main/rdf/vocab/dimensionvectors/VOCAB_QUDT-DIMENSION-VECTORS.ttl
+++ b/src/main/rdf/vocab/dimensionvectors/VOCAB_QUDT-DIMENSION-VECTORS.ttl
@@ -3998,14 +3998,14 @@ qkdv:A0E-2L2I0M1H-1T-2.5D0
   a qudt:QuantityKindDimensionVector_SI;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/dimensionvector>;
   rdfs:label "A0E-2L2I0M1H-1T-2.5D0" ;
-  qudt:dimensionExponentForAmountOfSubstance "0"^^xsd:float ;
-  qudt:dimensionExponentForElectricCurrent "-2"^^xsd:float ;
-  qudt:dimensionExponentForLength "2"^^xsd:float ;
-  qudt:dimensionExponentForLuminousIntensity "0"^^xsd:float ;
-  qudt:dimensionExponentForMass "1"^^xsd:float ;
-  qudt:dimensionExponentForThermodynamicTemperature "-1"^^xsd:float ;
-  qudt:dimensionExponentForTime "-2.5"^^xsd:float ;
-  qudt:dimensionlessExponent "0"^^xsd:float ;
+  qudt:dimensionExponentForAmountOfSubstance 0 ;
+  qudt:dimensionExponentForElectricCurrent -2 ;
+  qudt:dimensionExponentForLength 2 ;
+  qudt:dimensionExponentForLuminousIntensity 0 ;
+  qudt:dimensionExponentForMass 1 ;
+  qudt:dimensionExponentForThermodynamicTemperature -1 ;
+  qudt:dimensionExponentForTime -2.5 ;
+  qudt:dimensionlessExponent 0 ;
   qudt:hasReferenceQuantityKind quantitykind:ThermalInertia ;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/dimension> .
 
@@ -4013,14 +4013,14 @@ qkdv:A0E0L-6I0M1H0T-3D0
   a qudt:QuantityKindDimensionVector_SI;
   rdfs:label "A0E0L-6I0M1H0T-3D0";
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/dimensionvector>;
-  qudt:dimensionExponentForAmountOfSubstance "0"^^xsd:float ;
-  qudt:dimensionExponentForElectricCurrent "0"^^xsd:float ;
-  qudt:dimensionExponentForLength "-6"^^xsd:float ;
-  qudt:dimensionExponentForLuminousIntensity "0"^^xsd:float ;
-  qudt:dimensionExponentForMass "1"^^xsd:float ;
-  qudt:dimensionExponentForThermodynamicTemperature "0"^^xsd:float ;
-  qudt:dimensionExponentForTime "-3"^^xsd:float ;
-  qudt:dimensionlessExponent "0"^^xsd:float ;
+  qudt:dimensionExponentForAmountOfSubstance 0 ;
+  qudt:dimensionExponentForElectricCurrent 0 ;
+  qudt:dimensionExponentForLength -6 ;
+  qudt:dimensionExponentForLuminousIntensity 0 ;
+  qudt:dimensionExponentForMass 1 ;
+  qudt:dimensionExponentForThermodynamicTemperature 0 ;
+  qudt:dimensionExponentForTime -3 ;
+  qudt:dimensionlessExponent 0 ;
   qudt:hasReferenceQuantityKind quantitykind:SpectralRadiance ;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/dimension> .
 
@@ -4028,14 +4028,14 @@ qkdv:A0E0L-4I0M1H0T-3D0
 a qudt:QuantityKindDimensionVector_SI;
   rdfs:label "A0E0L-4I0M1H0T-3D0";
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/dimensionvector>;
-  qudt:dimensionExponentForAmountOfSubstance "0"^^xsd:float ;
-  qudt:dimensionExponentForElectricCurrent "0"^^xsd:float ;
-  qudt:dimensionExponentForLength "-4"^^xsd:float ;
-  qudt:dimensionExponentForLuminousIntensity "0"^^xsd:float ;
-  qudt:dimensionExponentForMass "1"^^xsd:float ;
-  qudt:dimensionExponentForThermodynamicTemperature "0"^^xsd:float ;
-  qudt:dimensionExponentForTime "-3"^^xsd:float ;
-  qudt:dimensionlessExponent "0"^^xsd:float ;
+  qudt:dimensionExponentForAmountOfSubstance 0 ;
+  qudt:dimensionExponentForElectricCurrent 0 ;
+  qudt:dimensionExponentForLength -4 ;
+  qudt:dimensionExponentForLuminousIntensity 0 ;
+  qudt:dimensionExponentForMass 1 ;
+  qudt:dimensionExponentForThermodynamicTemperature 0 ;
+  qudt:dimensionExponentForTime -3 ;
+  qudt:dimensionlessExponent 0 ;
   qudt:hasReferenceQuantityKind quantitykind:SpectralIrradiance , quantitykind:SpectralEmittance;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/dimension> .
 

--- a/src/main/rdf/vocab/dimensionvectors/VOCAB_QUDT-DIMENSION-VECTORS.ttl
+++ b/src/main/rdf/vocab/dimensionvectors/VOCAB_QUDT-DIMENSION-VECTORS.ttl
@@ -4007,7 +4007,7 @@ qkdv:A0E-2L2I0M1H-1T-2.5D0
   qudt:hasReferenceQuantityKind quantitykind:ThermalInertia ;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/dimension> .
 
-qkdv:A0E0L-6I0M1H0T-3D1
+qkdv:A0E0L-6I0M1H0T-3D0
   a qudt:QuantityKindDimensionVector_SI;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/dimensionvector>;
   qudt:dimensionExponentForAmountOfSubstance "0"^^xsd:float ;
@@ -4017,7 +4017,7 @@ qkdv:A0E0L-6I0M1H0T-3D1
   qudt:dimensionExponentForMass "1"^^xsd:float ;
   qudt:dimensionExponentForThermodynamicTemperature "0"^^xsd:float ;
   qudt:dimensionExponentForTime "-3"^^xsd:float ;
-  qudt:dimensionlessExponent "1"^^xsd:float ;
+  qudt:dimensionlessExponent "0"^^xsd:float ;
   qudt:hasReferenceQuantityKind quantitykind:SpectralRadiance ;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/dimension> .
 

--- a/src/main/rdf/vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL.ttl
+++ b/src/main/rdf/vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL.ttl
@@ -14899,39 +14899,39 @@ voag:QUDT-QUANTITY-KINDS-VocabCatalogEntry_v1.2
 
 # new quantity kinds
 
-quantitykind:spectralIrradiance
+quantitykind:SpectralIrradiance
   a qudt:QuantityKind ;
   rdfs:label "Measurement Unit of Spectral Irradiance";
   qudt:plainTextDescription "Spectral Irradiance is a radiometry term for the power per unit area of electromagnetic radiation per unit wavelength incident at a surface." ;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> ;
   qudt:hasDimensionVector qkdv:A0E0L-4I0M1H0T-3D0 .
 
-quantitykind:spectralEmittance
+quantitykind:SpectralEmittance
   a qudt:QuantityKind ;
   rdfs:label "Measurement Unit of Spectral Emittance";
   qudt:plainTextDescription "Spectral Emittance is a radiometry term for the power per unit area of electromagnetic radiation unit wavelength emerging from a surface." ;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> ;
   qudt:hasDimensionVector qkdv:A0E0L-4I0M1H0T-3D0 .
   
-quantitykind:spectralRadiance
+quantitykind:SpectralRadiance
   a qudt:QuantityKind ;
   rdfs:label "Measurement Unit of Spectral Radiance";
   qudt:plainTextDescription "Spectral Radiance is a radiometry term for the power per unit area of electromagnetic radiation per unit wavelength per unit solid angle emerging from a surface." ;
   qudt:hasDimensionVector qkdv:A0E0L-6I0M1H0T-3D-1;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> .
 
-quantitykind:thermalInertia
+quantitykind:ThermalInertia
   a qudt:QuantityKind ;
   rdfs:label "Measurement Unit of Thermal Inertia";
-  qudt:PlainTextDescription "Thermal inertia is a term used to describe the observed delays in a body's temperature response during heat transfers";
+  qudt:plainTextDescription "Thermal inertia is a term used to describe the observed delays in a body's temperature response during heat transfers";
   qudt:informativeReference "https://en.wikipedia.org/wiki/Thermal_inertia"^^xsd:anyURI ;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> ;
   qudt:hasDimensionVector qkdv:A0E-2L2I0M1H-1T-2.5D0.
   
-quantitykind:apparentThermalInertia
+quantitykind:ApparentThermalInertia
   a qudt:QuantityKind ;
   rdfs:label "Measurement Unit for a quantity approximately proportional to Thermal Inertia";
-  qudt:PlainTextDescription "Thermal inertia is a term used to describe the observed delays in a body's temperature response during heat transfers";
+  qudt:plainTextDescription "Thermal inertia is a term used to describe the observed delays in a body's temperature response during heat transfers";
   qudt:informativeReference "https://en.wikipedia.org/wiki/Thermal_inertia"^^xsd:anyURI ;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> ;
   qudt:hasDimensionVector qkdv:A0E0L0I0M0H-1T0D0 .

--- a/src/main/rdf/vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL.ttl
+++ b/src/main/rdf/vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL.ttl
@@ -14929,7 +14929,7 @@ quantitykind:SpectralRadiance
   a qudt:QuantityKind ;
   rdfs:label "Measurement Unit of Spectral Radiance";
   qudt:plainTextDescription "Spectral Radiance is a radiometry term for the power per unit area of electromagnetic radiation per unit wavelength per unit solid angle emerging from a surface." ;
-  qudt:hasDimensionVector qkdv:A0E0L-6I0M1H0T-3D0;
+  qudt:hasDimensionVector qkdv:A0E0L-1I0M1H0T-3D0;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> .
 
 quantitykind:ThermalInertia

--- a/src/main/rdf/vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL.ttl
+++ b/src/main/rdf/vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL.ttl
@@ -14897,3 +14897,41 @@ voag:QUDT-QUANTITY-KINDS-VocabCatalogEntry_v1.2
   rdfs:label "QUDT Quantity Kinds Vocabulary Catalog Entry v1.2" .
 
 
+# new quantity kinds
+
+quantitykind:spectralIrradiance
+  a qudt:QuantityKind ;
+  rdfs:label "Measurement Unit of Spectral Irradiance";
+  qudt:plainTextDescription "Spectral Irradiance is a radiometry term for the power per unit area of electromagnetic radiation per unit wavelength incident at a surface." ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> ;
+  qudt:hasDimensionVector qkdv:A0E0L-4I0M1H0T-3D0 .
+
+quantitykind:spectralEmittance
+  a qudt:QuantityKind ;
+  rdfs:label "Measurement Unit of Spectral Emittance";
+  qudt:plainTextDescription "Spectral Emittance is a radiometry term for the power per unit area of electromagnetic radiation unit wavelength emerging from a surface." ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> ;
+  qudt:hasDimensionVector qkdv:A0E0L-4I0M1H0T-3D0 .
+  
+quantitykind:spectralRadiance
+  a qudt:QuantityKind ;
+  rdfs:label "Measurement Unit of Spectral Radiance";
+  qudt:plainTextDescription "Spectral Radiance is a radiometry term for the power per unit area of electromagnetic radiation per unit wavelength per unit solid angle emerging from a surface." ;
+  qudt:hasDimensionVector qkdv:A0E0L-6I0M1H0T-3D-1;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> .
+
+quantitykind:thermalInertia
+  a qudt:QuantityKind ;
+  rdfs:label "Measurement Unit of Thermal Inertia";
+  qudt:PlainTextDescription "Thermal inertia is a term used to describe the observed delays in a body's temperature response during heat transfers";
+  qudt:informativeReference "https://en.wikipedia.org/wiki/Thermal_inertia"^^xsd:anyURI ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> ;
+  qudt:hasDimensionVector qkdv:A0E-2L2I0M1H-1T-2.5D0.
+  
+quantitykind:apparentThermalInertia
+  a qudt:QuantityKind ;
+  rdfs:label "Measurement Unit for a quantity approximately proportional to Thermal Inertia";
+  qudt:PlainTextDescription "Thermal inertia is a term used to describe the observed delays in a body's temperature response during heat transfers";
+  qudt:informativeReference "https://en.wikipedia.org/wiki/Thermal_inertia"^^xsd:anyURI ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> ;
+  qudt:hasDimensionVector qkdv:A0E0L0I0M0H-1T0D0 .

--- a/src/main/rdf/vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL.ttl
+++ b/src/main/rdf/vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL.ttl
@@ -14917,7 +14917,7 @@ quantitykind:SpectralRadiance
   a qudt:QuantityKind ;
   rdfs:label "Measurement Unit of Spectral Radiance";
   qudt:plainTextDescription "Spectral Radiance is a radiometry term for the power per unit area of electromagnetic radiation per unit wavelength per unit solid angle emerging from a surface." ;
-  qudt:hasDimensionVector qkdv:A0E0L-6I0M1H0T-3D1;
+  qudt:hasDimensionVector qkdv:A0E0L-6I0M1H0T-3D0;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> .
 
 quantitykind:ThermalInertia

--- a/src/main/rdf/vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL.ttl
+++ b/src/main/rdf/vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL.ttl
@@ -14917,7 +14917,7 @@ quantitykind:SpectralRadiance
   a qudt:QuantityKind ;
   rdfs:label "Measurement Unit of Spectral Radiance";
   qudt:plainTextDescription "Spectral Radiance is a radiometry term for the power per unit area of electromagnetic radiation per unit wavelength per unit solid angle emerging from a surface." ;
-  qudt:hasDimensionVector qkdv:A0E0L-6I0M1H0T-3D-1;
+  qudt:hasDimensionVector qkdv:A0E0L-6I0M1H0T-3D1;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> .
 
 quantitykind:ThermalInertia

--- a/src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl
+++ b/src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl
@@ -48546,7 +48546,7 @@ unit:W-PER-M2-MicroM-SR
   qudt:conversionMultiplier "1000000"^^xsd:decimel;
   qudt:plainTextDescription "A unit of spectral radiance that is the power radiating from a surface per unit of solid angle per unit of wavelength, measured in units of watts per meter squared per micrometer per steradian" ;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit>;
-  qudt:hasDimensionVector qkdv:A0E0L-6I0M1H0T-3D1;
+  qudt:hasDimensionVector qkdv:A0E0L-6I0M1H0T-3D0;
   qudt:hasQuantityKind quantitykind:SpectralRadiance .
 
 unit:J-PER-M2-Sec0pt5-K

--- a/src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl
+++ b/src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl
@@ -33979,7 +33979,7 @@ unit:MilliW-PER-M2-NanoM-SR
   qudt:conversionMultiplier 1000000.0 ;
   qudt:conversionMultiplierSN 1.0E6 ;
   qudt:hasDimensionVector qkdv:A0E0L-1I0M1H0T-3D0 ;
-  qudt:hasQuantityKind quantitykind:Unknown ;
+  qudt:hasQuantityKind quantitykind:spectralRadiance;
   qudt:symbol "mW/(m²·nm·sr)" ;
   qudt:ucumCode "mW.m-2.nm-1.sr-1"^^qudt:UCUMcs ;
   rdfs:isDefinedBy <http://qudt.org/$$QUDT_VERSION$$/vocab/unit> ;
@@ -38565,6 +38565,7 @@ unit:PER-K
   qudt:hasQuantityKind quantitykind:LinearExpansionCoefficient ;
   qudt:hasQuantityKind quantitykind:RelativePressureCoefficient ;
   qudt:hasQuantityKind quantitykind:ThermalExpansionCoefficient ;
+  qudt:hasQuantityKind quantitykind:apparentThermalInertia .
   qudt:iec61360Code "0112/2///62720#UAA193" ;
   qudt:symbol "/K" ;
   qudt:ucumCode "/K"^^qudt:UCUMcs ;
@@ -47785,7 +47786,7 @@ unit:W-PER-M2-NanoM-SR
   qudt:conversionMultiplier 1000000000.0 ;
   qudt:conversionMultiplierSN 1.0E9 ;
   qudt:hasDimensionVector qkdv:A0E0L-1I0M1H0T-3D0 ;
-  qudt:hasQuantityKind quantitykind:Unknown ;
+  qudt:hasQuantityKind quantitykind:spectralRadiance;
   qudt:symbol "W/(m²·nm·sr)" ;
   qudt:ucumCode "W.m-2.nm-1.sr-1"^^qudt:UCUMcs ;
   rdfs:isDefinedBy <http://qudt.org/$$QUDT_VERSION$$/vocab/unit> ;
@@ -48526,6 +48527,36 @@ unit:failures-in-time
   qudt:uneceCommonCode "FIT" ;
   rdfs:isDefinedBy <http://qudt.org/$$QUDT_VERSION$$/vocab/unit> ;
   rdfs:label "Failures In Time"@en .
+
+unit:W-PER-M2-MicroM
+  a qudt:unit ;
+  rdfs:label "Watts per meter squared micrometer";
+qudt:plainTextDescription "A unit of spectral emittance or spectral irradiance, which is the emittance or irradiance per unit of wavelength. It is watts per square meter per micrometer"@en-us ;
+  rdfs:seeAlso quantitykind:irradiance, quantitykind:emittance ;
+  qudt:hasQuantityKind quantitykind:spectralIrradiance, quantitykind:spectralEmittance ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit>;
+  qudt:hasDimensionVector qkdv:A0E0L-4I0M1H0T-3D0;
+  qudt:conversionMultiplierSM "1E6"^^xsd:double ;
+  qudt:conversionMultiplier "1000000"^^xsd:decimel.
+
+unit:W-PER-M2-MicroM-SR
+  rdfs:label "Watts per meter squared micrometer steradian";
+  a qudt:unit ;
+  qudt:conversionMultiplierSM "1E6"^^xsd:double ;
+  qudt:conversionMultiplier "1000000"^^xsd:decimel;
+  qudt:plainTextDescription "A unit of spectral radiance that is the power radiating from a surface per unit of solid angle per unit of wavelength, measured in units of watts per meter squared per micrometer per steradian" ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit>;
+  qudt:hasDimensionVector qkdv:A0E0L-6I0M1H0T-3D-1;
+  qudt:hasQuantityKind quantitykind:spectralRadiance .
+
+unit:J-PER-M2-Sec0pt5-K
+  rdfs:label "Joules per meter squared sqrt(second) degree kelvin";
+  qudt:hasQuantityKind quantitykind:thermalInertia ;
+  qudt:hasDimensionVector qkdv:A0E-2L2I0M1H-1T-2.5D0 ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit>;
+  qudt:plainTextDescription "A unit of thermal inertia measured as Joules per meter squared per square root Seconds per degree Kelvin ";
+  qudt:conversionMultiplierSM "1"^^xsd:double ;
+  qudt:conversionMultiplier "1"^^xsd:decimel.
 
 vaem:GMD_QUDT-UNITS-ALL
   a vaem:GraphMetaData ;

--- a/src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl
+++ b/src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl
@@ -48543,13 +48543,13 @@ unit:W-PER-M2-MicroM
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit>;
   qudt:hasDimensionVector qkdv:A0E0L-4I0M1H0T-3D0;
   qudt:conversionMultiplierSM "1E6"^^xsd:double ;
-  qudt:conversionMultiplier "1000000"^^xsd:decimel.
+  qudt:conversionMultiplier "1000000"^^xsd:decimal.
 
 unit:W-PER-M2-MicroM-SR
   rdfs:label "Watts per meter squared micrometer steradian";
   a qudt:unit ;
   qudt:conversionMultiplierSM "1E6"^^xsd:double ;
-  qudt:conversionMultiplier "1000000"^^xsd:decimel;
+  qudt:conversionMultiplier "1000000"^^xsd:decimal;
   qudt:plainTextDescription "A unit of spectral radiance that is the power radiating from a surface per unit of solid angle per unit of wavelength, measured in units of watts per meter squared per micrometer per steradian" ;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit>;
   qudt:hasDimensionVector qkdv:A0E0L-6I0M1H0T-3D0;
@@ -48562,7 +48562,7 @@ unit:J-PER-M2-Sec0pt5-K
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit>;
   qudt:plainTextDescription "A unit of thermal inertia measured as Joules per meter squared per square root Seconds per degree Kelvin ";
   qudt:conversionMultiplierSM "1"^^xsd:double ;
-  qudt:conversionMultiplier "1"^^xsd:decimel.
+  qudt:conversionMultiplier "1"^^xsd:decimal.
 
 vaem:GMD_QUDT-UNITS-ALL
   a vaem:GraphMetaData ;

--- a/src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl
+++ b/src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl
@@ -33981,7 +33981,7 @@ unit:MilliW-PER-M2-NanoM-SR
   qudt:conversionMultiplier 1000000.0 ;
   qudt:conversionMultiplierSN 1.0E6 ;
   qudt:hasDimensionVector qkdv:A0E0L-1I0M1H0T-3D0 ;
-  qudt:hasQuantityKind quantitykind:spectralRadiance;
+  qudt:hasQuantityKind quantitykind:SpectralRadiance;
   qudt:symbol "mW/(m²·nm·sr)" ;
   qudt:ucumCode "mW.m-2.nm-1.sr-1"^^qudt:UCUMcs ;
   rdfs:isDefinedBy <http://qudt.org/$$QUDT_VERSION$$/vocab/unit> ;
@@ -47792,7 +47792,7 @@ unit:W-PER-M2-NanoM-SR
   qudt:conversionMultiplier 1000000000.0 ;
   qudt:conversionMultiplierSN 1.0E9 ;
   qudt:hasDimensionVector qkdv:A0E0L-1I0M1H0T-3D0 ;
-  qudt:hasQuantityKind quantitykind:spectralRadiance;
+  qudt:hasQuantityKind quantitykind:SpectralRadiance;
   qudt:symbol "W/(m²·nm·sr)" ;
   qudt:ucumCode "W.m-2.nm-1.sr-1"^^qudt:UCUMcs ;
   rdfs:isDefinedBy <http://qudt.org/$$QUDT_VERSION$$/vocab/unit> ;

--- a/src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl
+++ b/src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl
@@ -38565,7 +38565,7 @@ unit:PER-K
   qudt:hasQuantityKind quantitykind:LinearExpansionCoefficient ;
   qudt:hasQuantityKind quantitykind:RelativePressureCoefficient ;
   qudt:hasQuantityKind quantitykind:ThermalExpansionCoefficient ;
-  qudt:hasQuantityKind quantitykind:apparentThermalInertia .
+  qudt:hasQuantityKind quantitykind:ApparentThermalInertia ;
   qudt:iec61360Code "0112/2///62720#UAA193" ;
   qudt:symbol "/K" ;
   qudt:ucumCode "/K"^^qudt:UCUMcs ;
@@ -48531,9 +48531,9 @@ unit:failures-in-time
 unit:W-PER-M2-MicroM
   a qudt:unit ;
   rdfs:label "Watts per meter squared micrometer";
-qudt:plainTextDescription "A unit of spectral emittance or spectral irradiance, which is the emittance or irradiance per unit of wavelength. It is watts per square meter per micrometer"@en-us ;
-  rdfs:seeAlso quantitykind:irradiance, quantitykind:emittance ;
-  qudt:hasQuantityKind quantitykind:spectralIrradiance, quantitykind:spectralEmittance ;
+  qudt:plainTextDescription "A unit of spectral emittance or spectral irradiance, which is the emittance or irradiance per unit of wavelength. It is watts per square meter per micrometer"@en-us ;
+  rdfs:seeAlso quantitykind:Irradiance, quantitykind:Emittance ;
+  qudt:hasQuantityKind quantitykind:SpectralIrradiance, quantitykind:SpectralEmittance ;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit>;
   qudt:hasDimensionVector qkdv:A0E0L-4I0M1H0T-3D0;
   qudt:conversionMultiplierSM "1E6"^^xsd:double ;
@@ -48547,11 +48547,11 @@ unit:W-PER-M2-MicroM-SR
   qudt:plainTextDescription "A unit of spectral radiance that is the power radiating from a surface per unit of solid angle per unit of wavelength, measured in units of watts per meter squared per micrometer per steradian" ;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit>;
   qudt:hasDimensionVector qkdv:A0E0L-6I0M1H0T-3D-1;
-  qudt:hasQuantityKind quantitykind:spectralRadiance .
+  qudt:hasQuantityKind quantitykind:SpectralRadiance .
 
 unit:J-PER-M2-Sec0pt5-K
   rdfs:label "Joules per meter squared sqrt(second) degree kelvin";
-  qudt:hasQuantityKind quantitykind:thermalInertia ;
+  qudt:hasQuantityKind quantitykind:ThermalInertia ;
   qudt:hasDimensionVector qkdv:A0E-2L2I0M1H-1T-2.5D0 ;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit>;
   qudt:plainTextDescription "A unit of thermal inertia measured as Joules per meter squared per square root Seconds per degree Kelvin ";

--- a/src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl
+++ b/src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl
@@ -48546,7 +48546,7 @@ unit:W-PER-M2-MicroM-SR
   qudt:conversionMultiplier "1000000"^^xsd:decimel;
   qudt:plainTextDescription "A unit of spectral radiance that is the power radiating from a surface per unit of solid angle per unit of wavelength, measured in units of watts per meter squared per micrometer per steradian" ;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit>;
-  qudt:hasDimensionVector qkdv:A0E0L-6I0M1H0T-3D-1;
+  qudt:hasDimensionVector qkdv:A0E0L-6I0M1H0T-3D1;
   qudt:hasQuantityKind quantitykind:SpectralRadiance .
 
 unit:J-PER-M2-Sec0pt5-K

--- a/src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl
+++ b/src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl
@@ -48565,7 +48565,7 @@ unit:MicroW-PER-CentiM2-MicroM-SR
   qudt:hasDimensionVector qkdv:A0E0L-1I0M1H0T-3D0;
   qudt:hasQuantityKind quantitykind:SpectralRadiance .
 
-unit:J-PER-M2-Sec0pt5-K
+unit:J-PER-M2-SEC0pt5-K
   rdfs:label "Joules per meter squared sqrt(second) degree kelvin";
   qudt:hasQuantityKind quantitykind:ThermalInertia ;
   qudt:hasDimensionVector qkdv:A0E-2L2I0M1H-1T-2.5D0 ;

--- a/src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl
+++ b/src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl
@@ -48552,7 +48552,17 @@ unit:W-PER-M2-MicroM-SR
   qudt:conversionMultiplier "1000000"^^xsd:decimal;
   qudt:plainTextDescription "A unit of spectral radiance that is the power radiating from a surface per unit of solid angle per unit of wavelength, measured in units of watts per meter squared per micrometer per steradian" ;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit>;
-  qudt:hasDimensionVector qkdv:A0E0L-6I0M1H0T-3D0;
+  qudt:hasDimensionVector qkdv:A0E0L-1I0M1H0T-3D0;
+  qudt:hasQuantityKind quantitykind:SpectralRadiance .
+
+unit:MicroW-PER-CentiM2-MicroM-SR
+  rdfs:label "Watts per centimeter squared micrometer steradian";
+  a qudt:unit ;
+  qudt:conversionMultiplierSM "1E4"^^xsd:double ;
+  qudt:conversionMultiplier "10000"^^xsd:decimal;
+  qudt:plainTextDescription "A unit of spectral radiance that is the power radiating from a surface per unit of solid angle per unit of wavelength, measured in units of watts per centimeter squared per micrometer per steradian" ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit>;
+  qudt:hasDimensionVector qkdv:A0E0L-1I0M1H0T-3D0;
   qudt:hasQuantityKind quantitykind:SpectralRadiance .
 
 unit:J-PER-M2-Sec0pt5-K

